github/billing.go

```diff
@@ -0,0 +1,129 @@
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	actionsOrgBilling := new(ActionBilling)
+	resp, err := s.client.Do(ctx, req, actionsOrgBilling)
+	return actionsOrgBilling, resp, err
+}
+
+// GetPackagesBillingOrg returns the free and paid storage used for GitHub Packages in gigabytes for an Org.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/billing#get-github-packages-billing-for-an-organization
+func (s *BillingService) GetPackagesBillingOrg(ctx context.Context, org string) (*PackageBilling, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/settings/billing/packages", org)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	packagesOrgBilling := new(PackageBilling)
+	resp, err := s.client.Do(ctx, req, packagesOrgBilling)
+	return packagesOrgBilling, resp, err
+}
+
+// GetStorageBillingOrg returns the estimated paid and estimated total storage used for GitHub Actions
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/billing#get-shared-storage-billing-for-an-organization
+func (s *BillingService) GetStorageBillingOrg(ctx context.Context, org string) (*StorageBilling, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/settings/billing/shared-storage", org)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	storageOrgBilling := new(StorageBilling)
+	resp, err := s.client.Do(ctx, req, storageOrgBilling)
+	return storageOrgBilling, resp, err
+}
+
+// GetActionsBillingUser returns the summary of the free and paid GitHub Actions minutes used for a user.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/billing#get-github-actions-billing-for-a-user
+func (s *BillingService) GetActionsBillingUser(ctx context.Context, user string) (*ActionBilling, *Response, error) {
+	u := fmt.Sprintf("users/%v/settings/billing/actions", user)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	actionsUserBilling := new(ActionBilling)
+	resp, err := s.client.Do(ctx, req, actionsUserBilling)
+	return actionsUserBilling, resp, err
+}
+
+// GetPackagesBillingUser returns the free and paid storage used for GitHub Packages in gigabytes for a user.
+//
+// GitHub API docs: https://docs.github.com/en/rest/reference/billing#get-github-packages-billing-for-an-organization
+func (s *BillingService) GetPackagesBillingUser(ctx context.Context, user string) (*PackageBilling, *Response, error) {
+	u := fmt.Sprintf("users/%v/settings/billing/packages", user)
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+	packagesUserBilling := new(PackageBilling)
+	resp, err := s.client.Do(ctx, req, packagesUserBilling)
+	return packagesUserBilling, resp, err
+}
+
+// GetStorageBillingUser returns the estimated paid and estimated total storage used for GitHub Actions
```

Comment:
Same here.

Reply:
your suggestion was https://github.com/google/go-github/pull/1797#discussion_r571637219

Reply:
Yes, that link is correct.  It has two lines in it:

```
// GetStorageBillingUser returns the estimated paid and estimated total storage used for GitHub Actions
// and GitHub Packages in gigabytes for a user.
```

Reply:
I think now it is ok @gmlewis and again sorry for the force push